sei_dbt/README.md

```diff
@@ -79,17 +108,155 @@ Then, it will:
          path: "path/to/your/database.db"  # Path to your DuckDB database file
          extensions: ['parquet', 'json']   # Optional extensions to load, if needed
             duckdb_project:
+               threads: 4
+         external_root: "data/dev"
 
        test:
          type: duckdb
          path: "test.db"  # Test database path
+         external_root: "data/test" 
          
        prod:
          type: duckdb
          path: "prod.db"  # Production database path
+         external_root: "data/prod"
    ```
+   
+In this `profiles.yml`:
```

Comment:
This section should be written less generally, and more with the specific use-case of this repo in mind. What type of profiles.yml is needed for someone working with this repo? Do they really need test/prod/dev database paths? Where should they store it?

Reply:
Agree. Will make the changes.